lib/price/provider.go

```diff
@@ -0,0 +1,13 @@
+package price
+
+import (
+	"context"
+
+	"github.com/omni-network/omni/lib/tokens"
+)
+
+// Provider is the price provider interface.
+type Provider interface {
+	// GetPriceUSD returns the price of each provided token in USD.
+	GetPriceUSD(ctx context.Context, tokens ...tokens.Token) (map[tokens.Token]float64, error)
```

Comment:
IMHO, a Get prefix is not needed here. I believe `Price` reads much better. Also, do we every expect other currency other than USD? If yes, the currency needs to be a parameter and the method will probably be called `Convert` and the interface `Converter`. If we are unlikely to use a different currency, I'd just leave it `Pricer` and `Price`.